routes/web.php

```diff
@@ -1,84 +1,24 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
-use App\Http\Controllers\TaskController;
-#P.54
-use App\Models\Task;
-use Illuminate\Http\Request;
-use Illuminate\Support\Facades\Validator;
-
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider and all of them will
-| be assigned to the "web" middleware group. Make something great!
-|
-*/
+use App\Http\Controllers\{
+    TaskController,
+    UserController,
+    AuthenticationController
+};
 
 Route::get('/', function () {
-    return view('welcome');
+    return to_route('tasks.index');
 });
-// タスク一覧
-Route::get('/tasks', [TaskController::class, 'index']);
-// タスクの保存
-Route::post('/task',[TaskController::class,'store']);
-// タスクの削除     Controllerのclass,Controllerの中にあるメソッドを指定する
-Route::delete('/task/{task}',[TaskController::class,'destroy']);
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-// #タスク一覧表示
-// Route::get('/',function(){
-//     // Task:: orderByでcreated_atをソート、getで取得
-//     $tasks = Task::orderBy('created_at','asc')->get();
-
-//     return view('tasks', [
-//         'tasks' => $tasks
-//     ]);
-// });
-// #新タスク追加
-// Route::post('/task',function(Request $request){
-//     #引数でPOSTデータを取得（use Illuminate\Http\Request;を記述し忘れるとエラー）
-
-//     // 第1引数:バリデーションを行うデータ
-//     // 第2引数:バリデーションルール
-//     $validator = Validator::make($request->all(),[
-//         'name' => 'required|max:255',
-//     ]);
-
-//     if($validator->fails()){
-//         return redirect('/')
-//         ->withInput()   //エラーをセッションに保存 全Viewから$errorsで見れるようになる
-//         ->withErrors($validator);   //すべての内容をセッションに保存
-//     }
 
-//     $task = new Task;
-//     // inputタグのname属性からタスク名を受け取る
-//     $task->name = $request->name;
-//     $task->save();
+Route::get('/tasks', [TaskController::class, 'index'])->name('tasks.index');
+Route::post('/task',[TaskController::class,'store'])->name('tasks.store');
+Route::delete('/tasks/{task}',[TaskController::class,'destroy'])->name('tasks.destroy');
 
-//     return redirect('/');
-// });
-// #POSTのリクエストをDELETEに見せかける記述
-// // tasksテーブルのIDが６であれば、変数$taskに６が入る
-// Route::delete('/task/{task}',function(Task $task){
-//     // /{task}でタスク名が入る。testって指定ならtestっていうタスクが入る
-//     $task->delete();
-    
-//     // return redirect('/');
-// });
+Route::get('/authentications/create',[AuthenticationController::class,'create'])->name('authentications.create');
+Route::post('/authentications',[AuthenticationController::class,'store'])->name('authentications.store');
+Route::get('/authentications/complete/{token}',[AuthenticationController::class,'complete'])->name('authentications.complete');
 
+Route::get('/users/create/{token}',[UserController::class,'create'])->name('users.create');
```

Comment:
```suggestion
Route::get('/users/create/{authentication_token}',[UserController::class,'create'])->name('users.create');
```
(token)がなにが入るからわからないので、authenticationsテーブルのtokenカラムの値ということが
わかるようにしてください。

Reply:
ご指摘、ご説明ありがとうございます。
authentication_tokenで統一いたしました。
ご確認よろしくお願いいたします。
routes/web.php